Sources/NIOCore/NIOSendable.swift

```diff
@@ -23,3 +23,19 @@ public typealias NIOSendable = Any
 #else
 public protocol NIOPreconcurrencySendable {}
 #endif
+
+#if swift(>=5.5) && canImport(_Concurrency)
+@usableFromInline
+struct UnsafeTransfer<Wrapped> : @unchecked Sendable {
```

Comment:
nit:
```suggestion
struct UnsafeTransfer<Wrapped>: @unchecked Sendable {
```

Reply:
Could you also add a small piece of documentation to the `struct` explaining what it's for?